### PR TITLE
Write rooms descriptions in run time & Adjust variable name

### PR DESCRIPTION
**Pull request by legendMK13**
1. changed variable 'level' used in gen_random_level, Room class, and in NPC functions to 'level_num' to prevent confusion. 'level' is used elsewhere to describe the graph of rooms.

2. changed the room descriptions such that shrt_desc only gives the landscape e.g. 'Dark Forest'. long_desc gives a sentence e.g. 'You are in a Dark Forest.' **However, the functions that return them are different**. get_shrt_desc returns the short description as is, but get_long_desc returns the room's long_desc, the contents' descriptions, and the shrt_desc of the adjacent rooms that have been seen.

_basically does a lot of what @BadScientist suggested._

TODO: set the shrt_desc and long_desc **variables** to show the room landscape

### CODE REVIEW

**Comment by legendMK13** [comment]
it can be pulled now or later. I'll still make a change maybe tomorrow or in a few hrs with the landscape changes
